server/tests/integration.rs

```diff
@@ -629,6 +641,14 @@ async fn error_if_user_doesnt_exist() {
     assert!(res.is_err());
 }
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+#[tracing::instrument(name = "Delete user test", skip_all)]
+async fn delete_user() {
+    let mut setup = TestBackend::single().await;
+    setup.add_user(&ALICE).await;
+    setup.delete_user(&ALICE).await;
+}
```

Comment:
Can we test whether the user is really deleted? Maybe trying to do a lookup and see it fail?

Reply:
Good point. Fixed in [2476a94](https://github.com/phnx-im/infra/pull/451/commits/2476a946802982fb18d6dc8843f06d305c08d7f0)